package.json

```diff
@@ -35,9 +40,21 @@
     "ads"
   ],
   "devDependencies": {
-    "grunt-contrib-concat": "~0.5.1",
-    "grunt": "~0.4.5",
-    "grunt-contrib-jshint": "^0.11.0",
-    "grunt-contrib-uglify": "~0.8.0"
+    "babel-core": "^6.26.0",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-stage-0": "^6.24.1",
+    "prettier": "^1.6.1",
+    "rollup": "^0.49.2",
+    "rollup-plugin-babel": "^3.0.2",
+    "rollup-plugin-filesize": "^1.4.2",
+    "rollup-plugin-license": "^0.5.0",
+    "rollup-plugin-uglify": "^2.0.1"
+  },
+  "xo": {
```

Comment:
is "xo" carried in any devDep so no need to add it?

Reply:
@thiamsantos xo is relatively new and can't expect it to be globally available like grunt or gulp, thus add it as dev dependency and update npm script to use from node_modules folder

Reply:
My bad, I only forgot to add it as Dev dependency. But you don't need to use it from the `node_modules` folder because the foi commands are available inside the bom scripts.